bench_decode_test.go

```diff
@@ -30,6 +30,7 @@ func NewClientStub(resp []byte) *ClientStub {
 		Dialer: func(ctx context.Context, network, addr string) (net.Conn, error) {
 			return stub.stubConn(initHello), nil
 		},
+		DisableIndentity: true,
```

Comment:
No, we don't want to send client setinfo on benchmark tests 😅

Reply:
I suspect the question about whether this is a type might be because `Indentity` is a very uncommon word (I'm not sure it's even a generally recognised word), whereas `Identity` would be much more common.